ci/images/ci-runner/hack/bin/copy-plnsvc-code.sh

```diff
@@ -4,6 +4,8 @@ set -o nounset
 set -o pipefail
 set -x
 
+echo "[DEBUG:PLR] This message must ALWAYS be seen in the PLR logs"
+
 echo "Copy source code of pipeline service to the ci-runner container"
 kubectl cp "./" "default/ci-runner:/source"
```

Comment:
this script is not needed, same as https://github.com/openshift-pipelines/pipeline-service/pull/760/files#r1318056837

Reply:
We can discuss the need for the script in the first comment. Resolving this one.